core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerClientPool.java

```diff
@@ -48,25 +48,37 @@ public final class BlockWorkerClientPool extends DynamicResourcePool<BlockWorker
       MetricKey.CLIENT_BLOCK_WORKER_CLIENT_COUNT.getName());
   private final AlluxioConfiguration mConf;
 
+  /**
+   * Create a block master client pool with default capacities.
+   *
+   * @param context Client context
+   * @param address Address of worker
+   * @param alluxioConf Alluxio Configuration
+   */
+  public BlockWorkerClientPool(ClientContext context,
```

Comment:
`alluxioConf` can be obtained from `context` so the 3rd parameter is redundant.

Reply:
Done